test/anchor-markdown-header.js

```diff
@@ -43,6 +43,9 @@ test('\ngenerating anchor in github mode', function (t) {
   , [ 'module-specific-variables-using-jsdoc-@module', null, '#module-specific-variables-using-jsdoc-module']
   , [ 'Jack & Jill', null, '#jack--jill']
   , [ 'replace – or —', null, '#replace--or-']
+  , [ 'Modules 📦', null, '#modules-']
```

Comment:
Is this correct to have a trailing `-` here?
Have you tried if github does it that way?